csi/crypto/crypto.go

```diff
@@ -10,17 +10,63 @@ import (
 
 const (
 	mapperFilePathPrefix = "/dev/mapper"
+
+	CryptoKeyDefaultCipher = "aes-xts-plain64"
+	CryptoKeyDefaultHash   = "sha256"
+	CryptoKeyDefaultSize   = "256"
+	CryptoDefaultPBKDF     = "argon2i"
```

Comment:
Assume these value the same as before to ensure the backward compatibility, right?

Reply:
Yes, These are default values or compatible on most distros.
only Debian 9 dose not support option "--pbdkf" because the cryptsetup version (1.7.4,  >2.0 needed)

Reply:
The encryption parameters are only required for initial encryption.

NIT: One could also leave the defaults blank then check whether the argument should be added to the call to luksFormat.
This way one can get rid of the Getter functions and one doesn't necessarily need the constructor function either.

But it's nit picking and the PR was good enough to get merged and testing started :)

Reply:
@mantissahz see comment above in regard to the removal of the defaults, to delegate to the luksformat call, that should solve your debian issue :)